modules/core/src/main/java/org/apache/synapse/rest/RESTUtils.java

```diff
@@ -153,4 +153,17 @@ private static void handleException(String msg, Throwable t) {
         throw new SynapseException(msg, t);
     }
 
+    /**
+     * Identify the API using the API context.
+     *
+     * @param path    request path
+     * @param context API context
+     */
+    public boolean getRESTApi(String path, String context) {
```

Comment:
this can be a static util method, can't it?

Reply:
Also I would think the method 'getRestApi' returns an api path, but apparently it returns a boolean indicating whether the the path is a rest api? Or what does it return? Shall we return the method accordingly?

Reply:
Fixed